soccer/src/gameplay/robocup-py.cpp

```diff
@@ -944,14 +943,18 @@ BOOST_PYTHON_MODULE(robocup) {
         .def("stay_behind_penalty_line", &GameState::stayBehindPenaltyLine)
         .def("is_our_restart", &GameState::isOurRestart)
         .def("get_ball_placement_point", &GameState::getBallPlacementPoint)
-        .def("get_goalie_id", &GameState::getGoalieId)
         .def("is_first_half", &GameState::isFirstHalf)
         .def("is_second_half", &GameState::isSecondHalf)
         .def("is_halftime", &GameState::isHalftime)
         .def("is_overtime1", &GameState::isOvertime1)
         .def("is_overtime2", &GameState::isOvertime2)
         .def("is_penalty_shootout", &GameState::isPenaltyShootout);
 
+    // TODO(Kyle): Propagate score to python correctly.
```

Comment:
Do we need issue numbers for this since it's part of the ROS transition?

Reply:
I think we should still do issue numbers

Reply:
Honestly, none of this python code is going to be reused.